students/Jean-Baptiste/lessons/lesson03/assignment03_solution_JB/basic_operations.py

```diff
@@ -0,0 +1,69 @@
+"""
+    This is to create a basic operations
+"""
+import logging
+import peewee
+from customers_model import Customer
+#'This is to add customer'
+def add_customer(customer_id, first_name, last_name, home_address, phone_number, email_address, status, credit_limit):
+    try:
+        new_customer = Customer.create(customer_id=customer_id, first_name=first_name, last_name=last_name, home_address=home_address, phone_number=phone_number, email_address=email_address, status=status, credit_limit=credit_limit)
+        logger.info("The new Customer is successfully added!")
+        new_customer.save()
+    except ValueError as e:
+        logging.info(e)
+        logging.info(f"Could not add {first_name} to db")
+    database.close()
+#'This is to serach customer'
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+logger.info('Working with Customer class to search, find')
+logger.info('Find and display by selecting with one Customer name...')
```

Comment:
These logging statements are useless if they are not in functions. They will only log when the file is imported or run, not when someone is actually searching for a customer.